frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/DistributionField.tsx

```diff
@@ -290,6 +290,47 @@ export function DistributionField(props: {
                 />
             </>
         )
+    } else if (props.cluster.distribution.upgradeInfo?.posthookDidNotRun) {
+        // CURATOR POSTHOOK JOB DID NOT RUN
+        return (
+            <>
+                <div>{props.cluster?.distribution.displayVersion}</div>
+                <AcmInlineStatus
+                    type={StatusType.danger}
+                    status={t('upgrade.upgradefailed', {
+                        version: props.cluster?.consoleURL
+                            ? ''
+                            : props.cluster?.distribution.upgradeInfo.desiredVersion,
+                    })}
+                    popover={
+                        props.cluster?.consoleURL
+                            ? {
+                                  headerContent: t('upgrade.upgradefailed', {
+                                      version: props.cluster?.distribution.upgradeInfo.desiredVersion,
+                                  }),
+                                  bodyContent: t('Upgrade posthook was not run', {
+                                      clusterName: props.cluster?.name,
+                                      version: props.cluster?.distribution.upgradeInfo.desiredVersion,
+                                  }),
+                                  footerContent: (
+                                      <a
+                                          href={
+                                              `${props.cluster?.consoleURL}/k8s/ns/${props.cluster.namespace}` +
+                                              '/cluster.open-cluster-management.io~v1beta1~ClusterCurator/' +
+                                              `${props.cluster.name}`
```

Comment:
I've used a link that targets the curator resource, as opposed to the cluster's ocp page, as the curator will have content from the job results.

Reply:
The `ClusterCurator` will always be created on the hub, so I'm not sure what the purpose of `consoleURL` is. Please try this out using a separate managed cluster instead of `local-cluster`.